test/timezone/lookup/test_google.rb

```diff
@@ -48,30 +48,48 @@ def test_google_request_denied_read_lat_long_coordinates
 
   def test_url_non_enterprise
     Timecop.freeze(Time.at(1_433_347_661)) do
-      result = lookup.send(:url, '123', '123')
+      mine = lookup('{ "status": "OK" }')
+
       params = {
         'location' => '123%2C123',
         'timestamp' => '1433347661',
         'key' => 'MTIzYWJj'
       }.map { |k, v| "#{k}=#{v}" }
 
-      assert_equal "/maps/api/timezone/json?#{params.join('&')}", result
+      url_method = Minitest::Mock.new
+      url_method.expect(
+        :call, "/maps/api/timezone/json?#{params.join('&')}", %w[123 123]
+      )
+
+      mine.stub :url, url_method do
```

Comment:
You still end up asserting on the private method call here so I don't see the benefit of this approach. The original spec (w/o the mocks) is easier to read.

Reply:
As you wish. Mocks is common practice. They're more complex in Minitest than in RSpec, yes.